### PR TITLE
Fulp T5 Trekkie Chems and Comebacks Rework

### PR DESCRIPTION
**Pull request by Saliferous**
## About The Pull Request
Attempts to fix several issues pertaining to tg's chem updates
Brings back Trekkie chems in a more balanced fashion along with adapting upon the T5 upgrade parts patch. Includes other throwback chems.

Adds new files: fulp_chem_CF_reagents.dm , fulp_chem_CF_medicines.dm
Edits file: chem_dispenser.dm to allow for an upgraded chem list past T4, pill.dm for charcoal add-on, cat2_medicine_reagents.dm for sanguibital desc edit
Adds reagents for Trekkie Chems and some others that allow chemistry to operate efficiently
NOTICE: Requires Xeon's T5 Upgrade Parts
The 'new' Trekkie Chems are Inject-Only, to limit the swarm of applications and to coerce players to use something other than patches or pills.
Also adds Charcoal back in but it acts as good-ole' charcoal instead of Multiver. 
Additionally, code for a replacement Synthflesh and Perfluorodecalin is included but commented out to prevent dupes. This is added for the intent to replace those chems once tg decides to wipe them out as well.
## Why It's Good For The Game
Tries to quite literally 'un-fuck' the catastrophe that is tg's latest update to chemistry by providing better endgame chem options with added T5 upgrade parts, will allow chemistry to streamline medical chems so they can focus on other objectives, and adding back in crucial chems that allow chemistry to function throughout the round. Also acts as a way of applying high-level research in another aspect, as an advanced society would have advanced medical chems. Just gotta use those hyposprays, Scotty!

## Changelog
:cl:
add: New Files: fulp_chem_CF_reagents , fulp_chem_CF_medicines
add: New reagents (Trekkie Chems: Bicaridine, Kelotane, AntiToxin, Tricordrazine)
add: New reagent (Trekamol: Activator required for Trekkie Chems)
add: New reagent (Charcoal: Adds it back in as regular charcoal, acts similar to multiver without the bull)
add: commented out reagents in fulp_chem_CF_reagents.dm
edit: pill.dm
edit: cat2_medicine_reagents.dm
edit: chem_dispenser.dm
edit: tgstation.dme
/:cl:

### CODE REVIEW

**Comment by CosmicPhoenix** [comment]
You need to add back Styptic Powder and Silver Sulf. They don't exist anymore.

**Comment by TheSwain** [comment]
Did they really remove styptic powder? What's the replacement now? How do you handle brute damage?

So all that's left I think is to move the item code out of code/modules/reagents/reagent_containers/pill.dm and into a custom .dm file, I think, and we'll be good to try this one out live.

**Comment by CosmicPhoenix** [comment]
Styptic Powder was replaced with Libital and the now replaced Sanguibital. Sanguibital was replaced with something that scales with your other damages so MDs will choke out their patients to heal faster with it.

File: code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	name = "Sanobital"
 	description = "A unique medicine that heals bruises, scaling with the rate at which one is bleeding out. Dilates blood streams, increasing the amount of blood lost. Overdosing further increases blood loss."
 	color = "#ECEC8D" // rgb: 236	236	141
-	taste_description = "whatever vampires would eat"
+	taste_description = "tastes like copper"
```
Comment (LLIkolnik) [inline code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm]: "tastes" shouldn't be here.

**Pull request merged**